syntax = "proto3";

package helloworld;


// The greeting service definition.
service Greeter {
  // Sends a string
  rpc SendString (SendRequest) returns (StringReply) {}

  // Sends a number (int)
  rpc SendNumber (SendRequest) returns (NumberReply) {}
}


// The request message (with no fields)
message SendRequest {}


// The response message containing the string
message StringReply {
  string message = 1;
}


// The response message containing the number (int)
message NumberReply {
  int32 number = 1;
}